osu.Framework/Bindables/ILeasedBindable.cs

```diff
@@ -11,7 +11,10 @@ public interface ILeasedBindable : IBindable
         /// <summary>
         /// End the lease on the source <see cref="Bindable{T}"/>.
         /// </summary>
-        void Return();
+        /// <returns>
+        /// Whether the lease was returned. Will be <c>false</c> if already returned.
```

Comment:
maybe slightly clearer?

```suggestion
        /// Whether the lease was returned by this call. Will be <c>false</c> if already returned.
```